web/screenshot/app.screenshot-spec.ts

```diff
@@ -0,0 +1,11 @@
+import { AppPage } from './app.po';
+import { browser} from 'protractor';
+
+describe('protractor-image-comparison desktop', () => {
+  it('should be the same as the baseline', async () => {
+    beforeEach(async () => {
+      await browser.get(browser.baseUrl);
+    });
```

Comment:
I'm not sure I understand the `beforeEach`. My understanding is that `beforeEach` is used outside the `it` functions. Is my understanding wrong? I agree that we want the `await browser.get(browser.baseUrl)` in the `it` function.

Reply:
I removed the `beforeEach` and just left the `await browser.get(browser.baseUrl)` in the `it` function. To clarify, when we add more tests to check different pages of the app, we would have to include a variant of the `await browser.get(browser.baseUrl.PAGE)` in that `it` function?

We don't want it in a `beforeEach` outside of the `it` function because each `it` function will be testing a different page so we wouldn't want the baseURL?

Reply:
Correct, when we add new test will have the `it` function for them to include the appropriate `await browser.get(...)`.